server/packets.py

```diff
@@ -480,9 +485,27 @@ def write_notification_packet(
 # FELLOW_SPECTATOR_JOINED = 42
 
 
+def write_fellow_spectator_joined_packet(user_id: int) -> bytes:
+    return write_packet(
+        packet_id=ServerPackets.FELLOW_SPECTATOR_JOINED,
+        packet_data_inputs=[
+            (DataType.I32, user_id),
+        ],
+    )
+
+
 # FELLOW_SPECTATOR_LEFT = 43
 
 
+def write_fellow_spectator_left_packet(user_id: int) -> bytes:
+    return write_packet(
+        packet_id=ServerPackets.FELLOW_SPECTATOR_LEFT,
+        packet_data_inputs=[
+            (DataType.I32, user_id),
+        ],
+    )
+
+
```

Comment:
While the `write_spectator_(joined|left)_packet` are for the host to get updates of who is spectating them, the "fellow" spectator packet variants are for when you're spectating someone, and there are other spectators - you will be sent these packets.